packages/SwingSet/docs/virtual-objects.md

```diff
@@ -154,6 +165,6 @@ Additional important details:
 
   `state.zot.push(4);`
 
-- A virtual object can be passed as a parameter in messages to other vats.  It will be passed by presence, just like any other non-data object you might send in a message parameter.
+- A VDO can be passed as a parameter in messages to other vats.  It will be passed by presence, just like any other non-data object you might send in a message parameter.
 
-- A virtual object's state may include references to other virtual objects. The latter objects will be persisted separately and only deserialized as needed, so "swapping in" a virtual object that references other virtual objects does not entail swapping in the entire associated object graph.
+- A VDO's state may include references to other VDOs. The latter objects will be persisted separately and only deserialized as needed, so "swapping in" a VDO that references other VDOs does not entail swapping in the entire associated object graph.
```

Comment:
Wow. Somehow I didn't catch that earlier and I didn't know that. It took me a moment to even realize how that could be possible. Cool!